source/_integrations/sonos.markdown

```diff
@@ -131,21 +131,34 @@ target:
   entity_id: media_player.sonos
 data:
   announce: true
-  media_content_type: "music"
-  media_content_id: "http://192.168.1.50:8123/local/sound_files/doorbell-front.mp3"
+  media_content_type: music
+  media_content_id: http://192.168.1.50:8123/local/sound_files/doorbell-front.mp3
```

Comment:
This change is actually the wrong way around. For more information on how to format YAML, check out our developers documentation, it has an extensive page on YAML formating.

Reply:
Ah, the new example had quotes in the URL, so this felt "cleaner". I'll adjust.